src/testing_system/cmd/test_mock_pipeline.py

```
@@ -0,0 +1,85 @@
+
+    questions = [
+        "Во сколько вылетает рейс SU-1231?"
+    ]
+
+    for q in questions:
+        print(f"User: {q}\n")
+        response = usecase.execute(question=q, system_prompt= """Ты помощник, который отвечает на вопросы ТОЛЬКО на основе предоставленных документов.
+
+Правила:
+1. Используй ТОЛЬКО информацию из документов выше
+2. Если в документах нет ответа, скажи: "У меня недостаточно информации для ответа"
```

Ruff (0.15.10)

[warning] String contains ambiguous `У` (CYRILLIC CAPITAL LETTER U). Did you mean `Y` (LATIN CAPITAL LETTER Y)?

(RUF001)